stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -35,4 +35,50 @@
         <item name="streamAvatarTextSize">@dimen/stream_avatar_text_size_large</item>
         <item name="streamAvatarOnlineIndicatorEnabled">true</item>
     </style>
-</resources>
\ No newline at end of file
+
+    <!--MessageInputViewNew-->
+    <declare-styleable name="StreamMessageInputView">
+        <!--Attachment Button-->
+        <attr name="showAttachmentButton" format="boolean" />
+        <attr name="attachmentButtonIcon" format="reference" />
+        <attr name="attachmentButtonIconColor" format="color|reference" />
+        <attr name="attachmentButtonIconPressedColor" format="color|reference" />
+        <attr name="attachmentButtonIconDisabledColor" format="color|reference" />
+        <attr name="attachmentButtonSelectedIconColor" format="color|reference" />
+        <attr name="attachmentButtonWidth" format="dimension|reference" />
+        <attr name="attachmentButtonHeight" format="dimension|reference" />
+        <attr name="attachmentCloseButtonBackground" format="reference" />
+
+        <!--Lightning Button-->
+        <attr name="showLightningButton" format="boolean" />
```

Comment:
We could name this something like `lightningButtonEnabled` to line up with the names of the related attributes, makes them easier to find